easybuild/easyconfigs/c/CrayCCE/CrayCCE-19.06.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayCCE'
+version = '19.06'
+
+homepage = 'https://pubs.cray.com/discover'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-cray
+(PE release: June 2019).\n"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+dependencies = [
+    # PrgEnv version is not pinned, as Cray recommends to use the latest
+    # (default) version
+    ('PrgEnv-cray', EXTERNAL_MODULE),
+    ('atp/2.1.3', EXTERNAL_MODULE),
+    ('cce/9.0.0-classic', EXTERNAL_MODULE),
+    # modules>3.2.11.1 fails with: 
+    # "Changing environment as dictated by module failed: invalid syntax"
+    ('modules/3.2.11.1', EXTERNAL_MODULE),
```

Comment:
default `modules` was changed today to `3.2.11.2` and we see this error again when loading `modules/3.2.11.1`

The reason is not completely clear, but it seems to be related to the python wrapper of the module tool: when switching to a different `modules` inside a module file, the python output is polluted with a bash statement :

```
perettig@dom101:~> module list
Currently Loaded Modulefiles:
  1) modules/3.2.11.2                                 12) job/2.2.4-7.0.0.1_3.16__g36b56f4.ari
  2) cce/9.0.0                                        13) dvs/2.11_2.2.129-7.0.0.1_5.1__g100a718a
  3) craype-network-aries                             14) alps/6.6.50-7.0.0.1_3.20__g962f7108.ari
  4) craype/2.6.0                                     15) rca/2.2.20-7.0.0.1_4.19__g8e3fb5b.ari
  5) cray-libsci/19.06.1                              16) atp/2.1.3
  6) udreg/2.3.2-7.0.0.1_4.15__g8175d3d.ari           17) perftools-base/7.1.0
  7) ugni/6.0.14.0-7.0.0.1_7.17__ge78e5b0.ari         18) PrgEnv-cray/6.0.5
  8) pmi/5.0.14                                       19) cray-mpich/7.7.8
  9) dmapp/7.1.1-7.0.0.1_5.4__g25e5077.ari            20) slurm/18.08.7-1
 10) gni-headers/5.0.12.0-7.0.0.1_7.18__g3b1768f.ari  21) craype-haswell
 11) xpmem/2.2.17-7.0.0.1_3.13__g7acee3a.ari


perettig@dom101:~> modulecmd python load daint-gpu CrayGNU/.19.03  | head
source /opt/cray/pe/modules/3.2.11.1/init/bash
os.environ['ALLINEA_QUEUE_DLL'] = '/opt/cray/pe/mpt/7.7.6/gni/mpich-gnu/7.1/lib/libtvmpich.so.3.0.1'
os.environ['ATP_HOME'] = '/opt/cray/pe/atp/2.1.3'
os.environ['ATP_IGNORE_SIGTERM'] = '1'
os.environ['ATP_MRNET_COMM_PATH'] = '/opt/cray/pe/atp/2.1.3/libexec/atp_mrnet_commnode_wrapper'
os.environ['ATP_POST_LINK_OPTS'] = '-Wl,-L/opt/cray/pe/atp/2.1.3/libApp/ '
os.environ['CRAYPE_DIR'] = '/opt/cray/pe/craype/2.5.18'
os.environ['CRAYPE_NETWORK_TARGET'] = 'aries'
os.environ['CRAYPE_VERSION'] = '2.5.18'
os.environ['CRAY_CPU_TARGET'] = 'haswell'
```